README.md

```diff
@@ -0,0 +1,11 @@
+#Overview
+
+This repository contains the code for all of our SLAM preliminary tests.
+
+##Stereo Cameras
+
+We attempt to use stereo cameras to determine depth and ultimately distance to visual objects.
+
+###Tools Used
+
+[StereoVision](https://github.com/erget/StereoVision) : OpenCV wrapper for simple calibration of stereo cameras. Contains wrapper modules as well as scripts for quick calibration.
```

Comment:
Perhaps you want to give instructions / snippets of code on how to calibrate with stereovision, since it's not directly clear how to do so.